packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -467,7 +467,7 @@ export const getAuthenticationStatusInfo = (account_status: GetAccountStatus): T
 export const mt5_community_url =
     'https://community.deriv.com/t/log-in-using-mt5-pc-or-mobile-app-application-guideline/49622';
 
-export const getFormattedJurisdictionCode = (jurisdiction_code: typeof Jurisdiction[keyof typeof Jurisdiction]) => {
+export const getFormattedJurisdictionCode = (jurisdiction_code?: typeof Jurisdiction[keyof typeof Jurisdiction]) => {
```

Comment:
why do we make this optional?

Reply:
`getEligibleAccountToMigrate` can return undefined if both conditions are false. so marked it as optional.
in case argument is not passed, it returns empty string anyway ⚡